demos/view-job-filters/README.md

```diff
@@ -0,0 +1,42 @@
+# Configure View Job Filters
+
+Basic configuration of the [View Job Filters plugin](https://plugins.jenkins.io/view-job-filters/)
+
+## Sample configuration
+
+```yaml
+jenkins:
+  views:
+    - list:
+        columns:
+        - "status"
+        - "weather"
+        - "jobName"
+        - "lastSuccess"
+        - "lastFailure"
+        - "lastDuration"
+        - "buildButton"
+        jobFilters:
+        - buildDurationFilter:
+            amount: "60.0"
+            amountTypeString: "Days"
+            buildCountTypeString: "Latest"
+            buildDurationMinutes: "5"
+            includeExcludeTypeString: "includeMatched"
+            lessThan: true
+        - buildStatusFilter:
+            building: false
+            inBuildQueue: true
+            includeExcludeTypeString: "includeMatched"
+            neverBuilt: true
+        - jobTypeFilter:
+            includeExcludeTypeString: "includeMatched"
+            jobType: "hudson.model.FreeStyleProject$DescriptorImpl@5d0507bc"
```

Comment:
I wouldn't say this is working, probably best to exclude it

Reply:
Sure :+1: 
I made the changes